webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -149,7 +145,7 @@ const ExperimentDetailsHeader: React.FC<Props> = (
         leftContent={<>
           <ExperimentState experiment={experiment} />
           <div className={css.experimentTitle}>
-            Experiment {experiment.id} <span>({experiment.name})</span>
+            Experiment {experiment.id} | <span>{experiment.name}</span>
```

Comment:
nit: maybe make the pipe the same color as the name instead of the same color as the id?